client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -1,5 +1,43 @@
-const ZoneCreator = () => {
-	return null;
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import page from 'page';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+
+const ZoneCreator = ( { siteSlug, translate } ) => {
+	const handleGoBack = () => {
+		page( `/extensions/zoninator/${ siteSlug }` );
+	};
+
+	return (
+		<div>
+			<HeaderCake onClick={ handleGoBack }>
+				{ translate( 'Add a zone' ) }
+			</HeaderCake>
+		</div>
+	);
 };
 
-export default ZoneCreator;
+ZoneCreator.propTypes = {
+	siteSlug: PropTypes.string,
+};
+
+const connectComponent = connect( state => {
+	return {
+		siteSlug: getSelectedSiteSlug( state ),
+	};
+} );
+
+export default flowRight(
+	connectComponent,
+	localize,
+)( ZoneCreator );
```

Comment:
With simple `connect` expressions as found here, I like to inline (and even drop `flowRight`):

```js
export default connect(
	state => ( {
		siteSlug: getSelectedSiteSlug( state )
	} )
)( localize( ZoneCreator ) );
```

I think that's a rather wide-spread style in Calypso. But up to you :slightly_smiling_face: